src/components/05_pages/Content/Content.js

```diff
@@ -175,9 +195,38 @@ class Content extends Component {
           </Button>
         </div>
 
+        <div className={styles.filters}>
+          {this.props.actions && (
+            <FormControl className={styles.formControl}>
+              <InputLabel htmlFor="actions">Actions</InputLabel>
+              <Select
+                value={this.state.action || ''}
+                onChange={e => {
+                  this.setState({ action: e.target.value });
+                }}
+                input={<Input name="action" id="action" />}
+                autoWidth
+              >
+                <MenuItem value="">
+                  <em>None</em>
+                </MenuItem>
+                {this.props.actions.map(action => (
+                  <MenuItem key={action.id} value={action.attributes.id}>
+                    {action.attributes.label}
+                  </MenuItem>
+                ))}
+              </Select>
+            </FormControl>
+          )}
+          {this.state.action && (
```

Comment:
Is there a better word than "Save"? Maybe apply

Reply:
I think `Apply` is definitely better than `Save`. Other options that pop into my mind would be `Execute` or just `Submit`.

Reply:
Thank you! I went with Apply, but ultimately the interaction model for VBO might change quite a bit anyway.